Orange/widgets/data/owdatainfo.py

```diff
@@ -179,10 +177,14 @@ def join_if(items):
             ("{} textual", str_metas)
         ))
 
+        if data.attributes:
+            self.data_attributes = pack_table(data.attributes.items())
```

Comment:
```py
else:
    self.data_attributes = ""
```
to clear this box for data without attributes.

Reply:
Long data attribute labels are now stripped. I decided against wrapping, as this would solve only the problem for shorter labels and or longer labels we would need to implement something entirely different.